career_development/ladder.md

```diff
@@ -13,102 +13,112 @@
 ## Dev Values : Simple, Test-Driven, Iterative, Communicative, Respectful, Skilled 
 
 ### Level 1
-
-#### simple 
--[ ] is working on naming variables and learning to extract methods
+#### Simple 
+-[ ] Is working on naming variables and learning to extract methods.
 	
-#### test-driven
--[ ] runs all tests before commiting, can understand where expectations are not met and fix some with the help of more senior developers
+#### Test-driven
+-[ ] Runs all tests before commiting.
+-[ ] Can understand where expectations are not met and fix some with the help of more senior developers.
 	
-#### iterative
--[ ] understands red-green-refactor
+#### Iterative
+-[ ] Understands red-green-refactor.
 	
-#### communication
--[ ] updates asana, comes prepared for standup, uses time tracking tools and vacation notifications
+#### Communication
+-[ ] Updates Asana.
+-[ ] Comes prepared for meetings.
+-[ ] Uses time tracking tools and vacation notifications.
 	
-#### respectful teammate
--[ ] on time for meetings, is enthusiastic
+#### Respectful teammate
+-[ ] On time for meetings.
+-[ ] Is enthusiastic.
```

Comment:
This is a bit controversial to me. It just smells a bit like we want people to not question stupid ideas etc. - even though I'm aware that it's not a all what it's meant to say.

I'm can't really come up with the perfect sentence, but here are some I came up with, maybe there are useful afterall:
- Is not lazy
- Puts in an effort
- Takes ownership

Reply:
Mmh. Definitely not what I had in mind. Let's find a better word.

Alternatives could be:
- `Has eagerness do to a good job`
- `Strives for excellence`
- `Cares about their work`.

Reply:
`Strives for excellence` is great. Maybe even `Strives for product excellence`